### DISCUSSION TITLE
Now there is a compatibility date in .kurviger

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#111] [Q&A] [answered]
After planning a short route in Cruiser I downloaded it as kurviger file and  tried to import it into the kurviger website. The result:
<img width="600" src="https://user-images.githubusercontent.com/121806238/229136426-913117b2-fae0-4743-a807-7d26e9d5bbfe.jpg"/>   
The Cruiser exported file: 
[20230331_153400.zip](https://github.com/devemux86/cruiser/files/11122411/20230331_153400.zip)
🤔

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report!

I see a different message?
Looks like they made the "info" section mandatory in their import parser.

It will be added in the next Cruiser update.

![Kurviger](https://user-images.githubusercontent.com/3484020/229140798-edcb5c47-c4c8-4ab5-96c5-c30416c588cc.jpg)

**Reply by 0709wiwiwi** [reply]
@WolfgangZ42 
Can you import a .kurviger file into Kurviger web without triggering a recalculation ?  
(Possible by the Kurviger app...so cold offer even (no recalc) TET design navigation.)

**Reply by devemux86** [reply]
Cruiser (and K1 / K2) always import the kurviger routes **offline**, exactly as designed by their creators.

**Comment by devemux86** [comment, marked as answer]
It should work properly in [Cruiser 3.0.16](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
Works perfect ... insane :-)

**Comment by ToToGSARider** [comment]
Hallo zusammen, heute die 1. Große Tour gefahren mit der Cruiser App mit einer Route aus kurviger geplant. Was soll ich sagen, es hat alles perfekt geklappt. Neuberechnung deaktiviert und bei Umleitungen mit WunderlinQ den Zoom genutzt. Alles wie früher bei K1 und K2. Meine Frau und ich sind absolut begeistert von dem Ergebnis heute mit der Cruiser App. Weiter so, freue mich schon wie es weitergeht mit der Anbindung an das WunderlinQ. 

Hello everyone, today the 1st Grand Tour driven with the Cruiser app with a route planned from curvy. What can I say, it all worked perfectly. Recalculation disabled and used the zoom on detours with WunderlinQ. Everything as before with K1 and K2. My wife and I are absolutely thrilled with the result today with the Cruiser app. Keep it up, looking forward to how it goes with the connection to the WunderlinQ.

**Reply by devemux86** [reply]
@ToToGSARider thanks for the kind words!

Development of Cruiser always continues and many new features are already planned for future versions.
